zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -79,12 +79,8 @@ angular.module('zeppelinWebApp')
   var angularObjectRegistry = {};
 
   var editorModes = {
-    'ace/mode/python': /^%(\w*\.)?(pyspark|python)\s*$/,
-    'ace/mode/scala': /^%(\w*\.)?spark\s*$/,
-    'ace/mode/r': /^%(\w*\.)?(r|sparkr|knitr)\s*$/,
     'ace/mode/sql': /^%(\w*\.)?\wql/,
-    'ace/mode/markdown': /^%md/,
-    'ace/mode/sh': /^%sh/
+    'ace/mode/markdown': /^%md/
```

Comment:
This PR reads syntax highlight language from `interpreter-setting.json` introduced by #835. I didn't remove this line because `%md`, `%*ql` is not following this new interpreter registration mechanism at the moment, in other words `%md`, `%*ql` interpreter doesn't have `interpreter-setting.json` file.
You can see the list of interpretesr that can apply new syntax highlight mechanism in ecb0af2